resources/certificate.rb

```diff
@@ -8,19 +8,22 @@
 property :keychain, String, required: true
 property :kc_passwd, String, required: true, sensitive: true
```

Comment:
I know already merged, just wondering about consistency in naming of "password" for `cert_password` vs. `cert_passwd` 🤔